doc/classes/GPUParticles3D.xml

```diff
@@ -34,7 +34,7 @@
 			<param index="3" name="custom" type="Color" />
 			<param index="4" name="flags" type="int" />
 			<description>
-				Emits a single particle. Whether [param xform], [param velocity], [param color] and [param custom] are applied depends on the value of [param flags]. See [enum EmitFlags].
+				Emits a single particle. Whether [param xform], [param velocity], [param color] and [param custom] are applied depends on the value of [param flags]. See [enum EmitFlags]. The default ParticleProcessMaterial will overwrite [param color] and use [param custom] as (rotation, age, animation, and lifetime).
```

Comment:
Sorry I didn't notice this earlier (was on phone when I commented), but upon reading now I actually disagree with the earlier suggestion to add "and". The parenthesis is meant to represent what scalar each coordinate of the `Color` type will map to. I suggest making it clearer by using code markup.
```suggestion
				Emits a single particle. Whether [param xform], [param velocity], [param color] and [param custom] are applied depends on the value of [param flags]. See [enum EmitFlags]. The default ParticleProcessMaterial will overwrite [param color] and map the coordinates of [param custom] to [code](rotation, age, animation, lifetime)[/code].
```